src/driver/uniformly_random_test_point_generator.py

```diff
@@ -37,19 +38,28 @@ def __init__(
         self,
         dataset: DataSet,
         rng: np.random.Generator,
+        npoints_generator: Callable[[int], int] = lambda x: None,
         npoints: int = MINIMUM_NUMBER_OF_TEST_POINTS,
     ):
-        """Returns a UniformlyRandomTestPointGenerator.
+        """Returns a LatinHypercubeRandomTestPointGenerator.
```

Comment:
I might misunderstand something, but I feel the original is correct 😅

Reply:
Thanks for catching this! I did too many copy pastes 😅